ansible/roles/rabbitmq/tasks/main.yml

```diff
@@ -1,29 +1,29 @@
 ---
 # Add and configure rabbitmq
 - name: Add rabbitmq official apt repository
-  sudo: true
+  become: true
```

Comment:
why become:true here and become:yes elsewhere (no action needed. existed before)

Reply:
`yes` and `true` evaluate to the same thing in yaml (== json `true`), so it doesn't matter which one you use. As to why we don't just pick one... I think we've just never bothered to agree on a repo-wide preference for one or the other.